MetaMask/metamask-extension
# fix: Update gas fee controller to v11.15.2

## **Description**

Updates the gas fee controller to v11.15.2, so that the gas fee controller is instantiated with the proper metadata.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24520?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/24383

## **Manual testing steps**

1. Start a tx
2. Add recipient and amount
3. Check console, there should not be a `Error: No metadata found for 'nonRPCGasFeeApisDisabled'` 

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
